app/kubernetes-ingress-controller/2.0.x/guides/using-udpingress.md

```diff
@@ -0,0 +1,351 @@
+---
+title: UDPIngress with Kong
```

Comment:
```suggestion
title: UDPIngress with Kong Gateway
```

Reply:
:+1: fixed in 1513aa547490817ee8ea05712cda81025c216b91